src/content/docs/browser/new-relic-browser/browser-pro-features/page-resources.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Automatically observe page resource assets'
+tags:
+  - Browser
+  - Browser monitoring
+  - Experimental features
+metaDescription: "Observes and reports on the performance of your webpages by automatically observing page resource timings."
+redirects:
+  - /docs/new-relic-browser/browser-pro-features/browser-features
+freshnessValidatedDate: never
+---
+
+[Resource Assets](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming) are reported natively by all major browser and enable you to observe and report on the performance of the assets your webpages import. New Relic Browser can automatically track these assets as `BrowserPerformance` events.
+
+<Callout variant="important">
+This feature is currently experimental and is only available for opt-in on copy/paste or NPM implementations of the agent. Please see [experimental features](/docs/browser/new-relic-browser/configuration/experimental-features) for more information on opting in to use experimental features. Please note that experimental features are subject to changes before general availability.
```

Comment:
- This feature is currently experimental and is only available for opt-in on copy/paste or NPM implementations of the agent. > This feature is currently experimental and is available only for opt-in on manual copy and paste, or NPM implementations of the agent. 
- Please note that experimental features....> Note that....